.expeditor/scripts/shared.ps1

```diff
@@ -72,15 +81,17 @@ function Install-RustToolchain($Toolchain) {
 function Wait-Supervisor($Timeout=1) {
   Write-Host "Waiting up to $Timeout seconds for Supervisor to start..."
   $startTime = [DateTime]::Now
-  $success = (Test-NetConnection -ComputerName localhost -Port 9631).TcpTestSucceeded
+  # Test-NetConnection could be used here, but it is not implemented in PowerShell core
+  # See https://github.com/PowerShell/PowerShell/issues/6932
+  $success = (Test-Connection -ComputerName localhost -TCPPort 9631).TcpTestSucceeded
```

Comment:
Unfortunately there is a `Test-Connection` in Windows powershell but it behaves slightly different. You could do something like:
```
If (Get-Command Test-NetConnection -ErrorAction SilentlyContinue) {
   Test-NetConnection ...
} else {
    Test-Connection ...
}
```